README.md

```diff
@@ -2,23 +2,35 @@
 
 ## Overview
 
-Abacus is a cross-chain communication system. It handles passing raw buffers
-between blockchains cheaply, and with minimal fuss. Like IBC and other
-cross-chain communication systems, Abacus creates channels between chains, and
-then passes its messages over the channel. Once a channel is established, any
-application on the chain can use it to send messages to any other chain.
+Abacus is a interchain messaging protocol. It handles passing generalized messages
+between blockchains cheaply, enabling genuine interchain applications. Inspired by IBC,
+Abacus creates channels between chains. In that respect, Abacus is a network between blockchains,
+and then passes messages between applications over these channels. Channels can be used 
+by numerous applications on any source or destination chain. Abacus is implemented wholly in
+smart contracts, and thus can be implemented in any turing complete smart contract chain.
 
-Compared to IBC and PoS light client based cross-chain communication, Abacus
-has weaker security guarantees. However, Abacus may be implemented on any smart
-contract chain, with no bespoke light client engineering. Because it does not run
-a light client, Abacus does not spend extra gas verifying remote chain block headers.
+Unfortunately, while Abacus might feel like magic, it isn't actually magic. Something's got to give.
```

Comment:
This feels a bit more casual then I would expect in a technical repo

Reply:
thats the goal ;)